kraken-app/kraken-app-controller/src/main/resources/application.yaml

```diff
@@ -193,6 +193,8 @@ app:
     endpoints:
       exposure:
         include: asset, component, product, ingestion, component-operation, component-mapper
+  push-activity-log-external:
```

Comment:
This flag controls  the feature of push-activity on/off. If we can add some  prefix, so that the similar feature flags can  be put together.
```yaml
app:
  features:
    push-activity-log-external:
      enable: true
```

Reply:
Hi @kuangxiang20240501 
"features" prefix added.